spec/features/vistors/visitor_cannot_see_classroom_content_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'visitor visits video show page' do
+  it 'Cannot see tutorials labeled classroom content' do
+    tutorial1 = create(:tutorial)
+    tutorial2 = create(:tutorial, description: "Amazing Much Learning!", classroom: true)
+
+      # video1 = create(:video, tutorial_id: tutorial1.id)
+      # video2 = create(:video, tutorial_id: tutorial1.id)
+      # video3 = create(:video, tutorial_id: tutorial2.id)
+      # video4 = create(:video, tutorial_id: tutorial2.id)
+
+      visit root_path
+
+      within(first('.tutorials')) do
+        expect(page).to have_content(tutorial1.title)
+        expect(page).to have_content(tutorial1.description)
+
+        expect(page).to_not have_content(tutorial2.title)
+        expect(page).to_not have_content(tutorial2.description)
+    end
+  end
```

Comment:
First test is testing to make sure the visitor cannot see tutorial 2 which has a classroom boolean value of true. Tutorial 1 did not need me to specify the boolean value because in the table it's defaulted to false.

Reply:
Tests look great!

Reply:
Love that sad path and happy path testing!